dotCMS/src/main/java/com/dotcms/security/apps/AppsAPIImpl.java

```diff
@@ -67,23 +68,23 @@
             .setVisibility(PropertyAccessor.FIELD, Visibility.ANY)
             .findAndRegisterModules();
 
-    private String toJsonString(final ServiceSecrets object) throws DotDataException {
+    private String toJsonString(final AppSecrets object) throws DotDataException {
         try {
             return jsonMapper.writeValueAsString(object);
         } catch (IOException e) {
             throw new DotDataException(e);
         }
     }
 
-    private ServiceSecrets readJson(final String json) throws DotDataException {
+    private AppSecrets readJson(final String json) throws DotDataException {
         try {
-            return jsonMapper.readValue(json, ServiceSecrets.class);
+            return jsonMapper.readValue(json, AppSecrets.class);
         } catch (IOException e) {
             throw new DotDataException(e);
         }
     }
 
-    private ServiceSecrets readJson(final char[] chars) throws DotDataException {
+    private AppSecrets readJson(final char[] chars) throws DotDataException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider using varargs for methods or constructors which take an array the last parameter.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5218098)